### ISSUE TITLE
[Protocol] StoreGeneratedPattern should not appear for Identity

### ISSUE THREAD

**Issue by karataliu** [OData/RESTier#116]
ModuleProducer.cs:

```
   MetadataProperty storeGeneratedPattern = null;
                    efProperty.MetadataProperties.TryGetValue(
                        AnnotationSchema + ":StoreGeneratedPattern",
                        true,
                        out storeGeneratedPattern);

                    if (storeGeneratedPattern != null)
                    {
                        SetComputedAnnotation(model, property);
                    }
```

This would generate StoreGeneratedPattern even the value is Identity.

Should verifiy the value is Computed when setting such pattern.

**Comment by karataliu** [comment]
```
<Term Type="Core.Tag" Name="Computed" AppliesTo="Property" DefaultValue="true">
<Annotation Term="Core.Description" String="A value for this property is generated on both insert and update"/>
</Term>
```

https://msdn.microsoft.com/en-us/library/system.data.metadata.edm.storegeneratedpattern(v=vs.110).aspx

**Comment by lewischeng-ms** [comment]
Resolved in #152
